swatch-product-configuration/build.gradle

```diff
@@ -4,11 +4,12 @@ plugins {
 }
 
 dependencies {
-    compileOnly platform(libraries["quarkus-bom"])
```

Comment:
Including quarkus-bom was for snakeyaml and slf4j.  Enforcing the quarkus-bom library was causing issues when trying to run the spring boot modules.  Opted to remove references to quarkus-bom and explicitly set the version number.